consensus-types/blocks/execution.go

```diff
@@ -40,8 +40,10 @@ func NewWrappedExecutionData(v proto.Message) (interfaces.ExecutionData, error)
 	case *enginev1.ExecutionBundleElectra:
 		// note: no payload changes in electra so using deneb
 		return WrappedExecutionPayloadDeneb(pbStruct.Payload)
+	case *enginev1.ExecutionBundleFulu:
+		return WrappedExecutionPayloadDeneb(pbStruct.Payload)
```

Comment:
isn't this the same as above? do we need case *enginev1.ExecutionBundleFulu and combine it with above?